src/app/container/container.component.ts

```diff
@@ -223,6 +223,15 @@ export class ContainerComponent extends Entry implements AfterViewInit, OnInit {
     }
   }
 
+  /**
+   * Select the Versions tab
+   *
+   * @memberof WorkflowComponent
+   */
+  public selectVersionsTab() {
+    this.selectTab(this.validTabs.indexOf('versions'));
```

Comment:
I have mixed feelings about this. I don't like unexpected navigation by a site, it can confuse me. And I don't think we do this sort of thing elsewhere in Dockstore.

On the other hand, it's not changing the whole page, nor the URL, just the active tab. And if as a user you wanted to address the issue right away, you would go to the versions tab, assuming you understood what a default version is.

Ideas:

1. Leave it as is
2. Don't do it at all
3. Update the dialog message saying you're going to be redirected to the versions tab.

Reply:
My rationale for switching to the version tab: to increase the chances that the user would successfully select a default version.  I'd recommend firing up this PR locally and trying it both ways, I feel like switching to the Versions tab reduces the friction quite a bit.

That said, I will implement it however the team wants...

Reply:
My comments were based on having tried it locally -- it hadn't clicked at first from reading the code that it would change my active tab, and then I saw it in action.

As I said, I have mixed feelings about it, not strong enough to actively oppose it -- I was proposing some options for consideration.